### ISSUE TITLE
How can i get draft's url?

### ISSUE THREAD

**Issue by marce1994** [release-drafter/release-drafter#326]
Hello everyone :)

I want to upload an asset to the final draft generated, there is any way to set an output with the draft url?

## what i have:

```yml
    - name: Create Release
      id: create_release
      uses: actions/create-release@v1.0.0
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        tag_name: ${{ github.ref }}
        release_name: Release ${{ github.ref }}
        draft: true
        prerelease: true
    - name: Upload Release Asset
      id: upload-release-asset 
      uses: actions/upload-release-asset@v1.0.1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
        asset_path: ./XX_Build.zip
        asset_name: XX_Build.zip
        asset_content_type: application/zip
```

## what i want:

```yml
    - uses: toolmantim/release-drafter@v5.2.0
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    - name: Upload Release Asset
      id: upload-release-asset 
      uses: actions/upload-release-asset@v1.0.1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
        asset_path: ./XX_Build.zip
        asset_name: XX_Build.zip
        asset_content_type: application/zip
```

**Comment by TimonVS** [comment]
This functionality is currently not supported by Release Drafter but should be trivial to add by logging the release draft url with some special formatting: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-output-parameter-set-output.

**Comment by dmyers** [comment]
Is there any way this could use the step output that way another step could use the release that is created? It would be really valuable for many workflows to be able to use that data.

One example is that GitHub does not trigger an event for draft releases so I can not listen for it and then trigger an API call to update the target branch which if I could get the release ID from the output of this I could then make that feature work.

**Comment by TimonVS** [comment]
Yup, we'll be exposing it as a step output. I'll get to this soon, seeing there are many people requesting this feature :)

**Comment by TimonVS** [comment]
This has been released in [v5.4.0](https://github.com/release-drafter/release-drafter/releases/tag/v5.4.0) :)

Be aware that `actions/upload-release-asset` doesn't overwrite release assets, so repeatedly running the workflow will result in failure. See https://github.com/release-drafter/release-drafter/pull/337#issuecomment-568505018 and https://github.com/release-drafter/release-drafter/pull/337#issuecomment-568634352 for more details. There's an open issue and PR that tackle this: https://github.com/actions/upload-release-asset/issues/13.
